modules/exploits/linux/http/freepbx_hotelwakeup_rce.rb

```diff
@@ -0,0 +1,120 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit4 < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'            => 'FreePBX < 13.0.188.1 Remote root exploit',
```

Comment:
I believe names are not supposed to have version numbers in here.  I would suggest `FreePBG Hotelwakeup RCE` or something along those lines